blackbox/docs/admin/cluster-management.rst

```diff
@@ -0,0 +1,55 @@
+.. _cluster_management:
+
+==================
+Cluster management
+==================
+
+With the ``crate-node`` command and the crate-node CLI tool, you can manage your
+CrateDB clusters by forming new clusters, repairing broken clusters, and broken
+nodes.
+
+The ``crate-node`` command
+--------------------------
+
+The ``crate-node`` command enables you to:
+
+ * Unsafely bootstrap a cluster
+ * Detach a node from its cluster
+ * Change the role of a node
+ * Eventually recover some data after a disaster
+
+Synopsis
+~~~~~~~~
+
+.. code-block:: console
+
+   bin/crate-node repurpose|detach-cluster|unsafe-bootstrap
+   [--ordinal <Integer>] [-C <KeyValuePair>]
+   [-h, --help] ([-s, --silent] | [-v, --verbose])
+
+Modes
+~~~~~
+
+This section introduces you to the ``crate-node`` command that is comprised of
+the three modes ``repurpose``, ``unsafe-bootstrap``, and ``detach cluster``.
+These modes allow you to execute all of the above CrateDB cluster management
+operations.
+
+* Use ``crate-node repurpose`` to delete data from a node if it used to be a
```

Comment:
perhaps these should be subheadings?

also, perhaps we should give one example use per mode

Reply:
All examples and how-tos can be found in the Troubleshooting wit the crate-node CLI section, that we want to link to. The subheading that leads the reader there can be found in the bottom of this doc.

Reply:
yes. but you only need one example

`crate-node repurpose` is already an example, but it's inline

we should list them as code blocks with `console` formatting and the `sh$` prefix

we should then explain what that command would do

for commands with additional options available to them (indicated in the command synopsis), we should explain that in the relevant subsection and give an example of it being used